Sources/SwiftProtobuf/Message+BinaryAdditions.swift

```diff
@@ -22,13 +22,17 @@ extension Message {
   /// - Parameters:
   ///   - partial: If `false` (the default), this method will check
   ///     `Message.isInitialized` before encoding to verify that all required
-  ///     fields are present. If any are missing, this method throws
+  ///     fields are present. If any are missing, this method throws.
   ///     `BinaryEncodingError.missingRequiredFields`.
+  ///   - options: The `BinaryEncodingOptions` to use.
   /// - Returns: A `SwiftProtobufContiguousBytes` instance containing the binary serialization
   /// of the message.
   ///
   /// - Throws: `BinaryEncodingError` if encoding fails.
-  public func serializedBytes<Bytes: SwiftProtobufContiguousBytes>(partial: Bool = false) throws -> Bytes {
+  public func serializedBytes<Bytes: SwiftProtobufContiguousBytes>(
+    partial: Bool = false,
+    options: BinaryEncodingOptions = BinaryEncodingOptions()
```

Comment:
I think for 1.x (if we pull this back), we'd have to have a method with and without this option rather than defaulting it.  Atleast I think that's why we have in some other things on the 1.x branch.

Reply:
Not opposed to this, but I'm curious as to what this would break considering the function is not overrideable or part of a protocol?

Reply:
This is a public method so changing its signature is an API breaking change. Users might be holding a partially applied call to this method and their signature will change and potentially break their code.